### ISSUE TITLE
2.0.0 breaks ui drastically

### ISSUE THREAD

**Issue by Faiyyaz** [LucazzP/responsive_widgets#3]
Hi,

It was good library before 2.0.0 now you have removed IconResponsive and getSize method too and after changing the library version in my project the ui looks very smaller than before.

May i know the reason for such drastic changes?

**Comment by LucazzP** [comment]
Hi!
So, the biggest problem I was having was that the texts didn't work with responsiveness, and because of that and some other small bugs that occurred in this responsiveness calculation, I preferred to leave the calculation to a widely used package and with that, I'm sure it works correctly, that is ScreenUtils.
The problem on the calculation of the responsiveness was that the package caught the width and the height and then make a scale number to resize the screen. This normally works well, but with the time I caught some overflows... The cause is because using 2 references to make the calculation, it makes it without precision and in some cases causes overflowing.
Besides that ScreenUtil uses only one reference, or the Height, or the Width, and the texts, a different calculation with the system.
So, the previous version works well, but you can find some bugs in those cases...
Basically the new version made the package an "extension" of the ScreenUtil, to use more easily on widgets, but don't replacing it.

**Comment by Faiyyaz** [comment]
Okay 
Thanks i will make the necessary changes and thanks for such a great library.
